lib/utils/blockchain-txs.ts

```diff
@@ -577,19 +577,14 @@ export const getNamePrice = async ({
   ensName: ENSName;
   durationInYears: bigint;
 }) => {
```

Comment:
It would be nice to have a try.. catch here

Reply:
The "register" function in line 300 is already handling the error

Reply:
I agree this function could return different result in case of error. 

The type aliasing can fool us at some point in the future.

A return type of `bigint` (is it returning bigint? I think so) `| null` would be ideal. If no price than return `null`, this gives minimun visibility for the outer function that some error happened in the price getter function.

For far future, error objects can be created to standardize errors handling, do not bother with this one line now.

Reply:
@FrancoAguzzi If we only return null when there is no price, the outer function won't know an error occurred. We should throw an error to ensure the outer function is aware of the issue